common/source/docs/common-mamba405-mk2.rst

```diff
@@ -0,0 +1,172 @@
+================================
+Mamba F405 MK2 Flight Controller
+================================
+
+.. image:: ../../../images/mamba_f405mk2_board.jpg
+     :target: ../_images/mamba_f405mk2_board.jpg
+
+The Mamba F405 MK2 is a flight controller produced by
+`Diatone <https://www.diatone.us>`__.
+
+
```

Comment:
We need to add a warning here that this autopilot does not have a BARO and currently cannot be used without an external BARO addition. I intend to move SpeedyBee, this, and the other autopilot to a special section of the autopilot landing page since they need external baros